src/api/status/public/index.html

```diff
@@ -51,20 +51,26 @@
           aria-hidden="true"
           id="iconSidenav"
         ></i>
-        <a
-          class="navbar-brand m-0"
-          href=" https://demos.creative-tim.com/material-dashboard/pages/dashboard "
-          target="_blank"
-        >
-          <img src="assets/img/logo-ct.png" class="navbar-brand-img h-100" alt="main_logo" />
-          <span class="ms-1 font-weight-bold text-white">Material Dashboard 2</span>
+        <a class="navbar-brand m-0" href="./" target="_blank">
+          <img src="./logo.svg" class="navbar-brand-img h-100" alt="main_logo" />
+          <span class="ms-1 font-weight-bold text-white">Telescope Dashboard</span>
         </a>
       </div>
       <hr class="horizontal light mt-0 mb-2" />
-      <div class="collapse navbar-collapse w-auto max-height-vh-100" id="sidenav-collapse-main">
+      <div class="w-auto max-height-vh-100" id="sidenav-collapse-main">
         <ul class="navbar-nav">
           <li class="nav-item">
-            <a class="nav-link text-white active bg-gradient-primary" href="./pages/build.html">
+            <a class="nav-link text-white active" href="./">
+              <div
+                class="text-white text-center me-2 d-flex align-items-center justify-content-center"
+              >
+                <i class="material-icons opacity-10">dashboard</i>
+              </div>
+              <span class="nav-link-text ms-1">Dashboard</span>
+            </a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link text-white" href="./pages/Build.html">
```

Comment:
This is 404'ing for me.  Should be ./pages/build.html (lower case b)

Reply:
Edited!